modules/indexing/src/test/java/org/apache/ignite/client/FunctionalQueryTest.java

```diff
@@ -237,6 +237,39 @@ public void testGettingEmptyResultWhenQueryingEmptyTable() throws Exception {
         }
     }
 
+    /** */
+    @Test
+    public void testMixedQueryAndCacheApiOperations() throws Exception {
+        try (Ignite ignored = Ignition.start(Config.getServerConfiguration());
+             IgniteClient client = Ignition.startClient(
+                 new ClientConfiguration().setBinaryConfiguration(new BinaryConfiguration().setCompactFooter(true))
+                     .setAddresses(Config.SERVER))
+        ) {
+            String cacheName = "PersonCache";
```

Comment:
It is not obvious for even many IgniteDevelopers that key and id field in Person object
are different things. Do you think that probably it is better to use another class in this test with additional field, that is not mentioned in DDL and doesn'thave same name and type as key?

This is up to you, of course, but I'd rather do it that way.

Reply:
I've renamed `id` column to `key`